src/test/scala/nl.knaw.dans.easy.ccw/CollectSpec.scala

```diff
@@ -34,11 +35,12 @@ class CollectSpec extends TestSupportFixture with BeforeAndAfterAll {
   val datamanagerProperties = configuration.datamanagers
   val commonCurationArea = testDir / "easy-common-curation-area"
   val datamanagerCurationAreas = testDir / "datamanager-curation-areas"
-  val managerCurationDirString = datamanagerCurationAreas / "$unix-user/curation-area" toString
+  val managerCurationDir = datamanagerCurationAreas / "$unix-user/curation-area" toString
+  def managerCurationArea(datamanager: DatamanagerId) = File(managerCurationDir.replace("$unix-user", datamanager))
   val jannekesCurationArea = datamanagerCurationAreas / "janneke/curation-area"
   val testLog = testDir / ".." / "test.log"
 
-  val collector = new EasyCollectCurationWorkApp(commonCurationArea, managerCurationDirString, datamanagerProperties)
+  val collector = new EasyCollectCurationWorkApp(commonCurationArea, managerCurationArea, datamanagerProperties)
```

Comment:
wouldn't it be easier to do `_ => managerCurationDir`, with `managerCurationDir` having no placeholders?

Reply:
Perhaps I don't quite understand what you mean, but also in the test the replacement has to take place (all the datamanagers are scanned through in EasyCollectCurationWorkApp)

Reply:
If you would just give a directory as a result of this function (thus no replacement), then all the datamanagers would get this same directory as their private curation area.

Reply:
ah, I see, so this 'fake' function doesn't work. Then your solution might be the easiest indeed... 👍